### ISSUE TITLE
Script does nothing but printing "Hello World"

### ISSUE THREAD

**Issue by htgoebel** [aaronst/macholibre#3]
When installing with `pip install …`, the command `macholibre` Print "Hello World" and does nothing.

This is caused be setup.py defining the entry-point to be `macholibre:main`, which just print Hello World.

**Comment by aaronst** [comment]
Hi @htgoebel,

Sorry for taking a while to respond.  I'm a little confused by your question, can you show me exactly how you're trying to run `macholibre` on the command line?

I get the following output:

```
macholibre/src/macholibre$ python macholibre.py
MachoLibre - Universal & Mach-O Binary Parser
Usage: python macholibre.py (-r <directory|glob> | <file>) [options]
Options:
-h         -        Show this help stuff
-o     <output>     Specify output file
-r <directory|glob> Parse directory or glob

For single files, make sure you put the filename first before any options.
Also, globs need to be in quotes

Examples:

(Single File)
python macholibre.py otool -o otool.json

(Glob)
python macholibre.py -r "machos/*" -o machos.json
```

**Comment by htgoebel** [comment]
I'm not using the `macholibre.py` file, but the script `macholibre` which gets installed when installing this package using `pip install macholibre`.
```
$ virtualenv /tmp/test
…
$ /tmp/test/bin/pip install git+https://github.com/aaronst/macholibre.git
…
$ /tmp/test/bin/macholibre --help
Hello World
```

**Comment by aaronst** [comment]
I *think* this is fixed now, let me know.

**Comment by htgoebel** [comment]
Yes, this is fixed by abdb9c9a4378a1ff261525bbb75d7062eff95e5b. Thanks.
